tests/jerry/private_fields.js

```diff
@@ -327,3 +327,22 @@ class O {
 var var16 = new O();
 var16.b();
 assert(var16.c() == 12);
+
+// Private fields are accessible in Array object
+class C3 extends Array{
+  #a=1;
+  b(){
+    return this.#a;
+   }
+}
+var var17 = new C3();
+assert(var17.b()==1);
+
+class C4 extends Array{
+  #a(){return 1}; 
```

Comment:
style:
- no space between ) {
- return 1 should be on a separate line
- missing ; after return 1
- extra ; after }